tape/src/main/java/com/squareup/tape2/QueueFile.java

```diff
@@ -581,8 +673,8 @@ public void clear() throws IOException {
 
     if (zero) {
       // Zero out data.
-      raf.seek(HEADER_LENGTH);
-      raf.write(ZEROES, 0, INITIAL_LENGTH - HEADER_LENGTH);
+      raf.seek(headerLength);
```

Comment:
That won't be true when we're done. If legacy format isn't forced queues will be upgraded.